src/js/search.js

```diff
@@ -20,7 +20,7 @@ search.addIndex('services');
 search.addIndex('name (old)');
 search.addIndex('email');
 search.addIndex('pops_served');
-search.addIndex('wheelchair');
+search.addIndex('wheelchair')
```

Comment:
Interesting, what's the thinking here? Another indication that we need to set up linting. I will submit a PR for that.